src/hooks/useMultiSelect.ts

```diff
@@ -0,0 +1,87 @@
+import { useState } from "react";
+import {
+  MultiSelectItem,
+  SelectedItems,
+} from "../components/MultiSelect/MultiSelectTypes";
+
+export default function useMultiSelect(multiSelectId, items) {
```

Comment:
This needs an interface for the two props.

Add a comment block to describe this hook:
```
/**
 * description ....
 */
export default function useMultiSelect(....
```